lib/components/SuperCombo.js

```diff
@@ -300,6 +309,9 @@ function MLSuperCombo$update() {
 
     var arrToShow = this._filteredOptionsData.slice(this._startIndex, this._endIndex);
 
+    const hasTooltips = arrToShow && Array.isArray(arrToShow) && !!arrToShow.find(v => v.item);
```

Comment:
same as above
is check value redundant as Array.isArray can catch undefined, if that is the case.